cves/CVE-2019-11358.yml

```diff
@@ -141,8 +170,13 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    no mention of specifications were mentioned in the documentation. This
+    seems to come from an internal jQuery issue, and, therefore, this
```

Comment:
Too, many, commas, consider refactoring "internal jQuery issue, and, therefore, this"